Enable us to attempt builds of post-1.17 images, until upstream update their
constraints.
---
diff --git a/fluent-plugin-label-router.gemspec b/fluent-plugin-label-router.gemspec
index 7e29ed1..50b080a 100644
--- a/fluent-plugin-label-router.gemspec
+++ b/fluent-plugin-label-router.gemspec
@@ -24,5 +24,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "test-unit", "~> 3.0"
   spec.add_dependency "prometheus-client", ">= 2.1.0"
   spec.add_dependency "concurrent-ruby"
-  spec.add_runtime_dependency "fluentd", [">= 1.17.1", "< 1.18"]
+  spec.add_runtime_dependency "fluentd", ">= 1.17.1"
 end
